lib/SymlinkPlugin.js

```diff
@@ -23,13 +23,18 @@ module.exports = class SymlinkPlugin {
 
 			let containsSymlink = false;
 			forEachBail.withIndex(paths, (path, idx, callback) => {
+				if(/^[a-zA-Z]:$/.test(path)) {
+					// 'F:' resolves to the current working directory of drive 'F' if the current working directory is a symlink.
+					path += "\\";
+				}
 				fs.readlink(path, (err, result) => {
 					if(!err && result) {
 						pathSeqments[idx] = result;
 						containsSymlink = true;
 						// Shortcut when absolute symlink found
-						if(/^(\/|[a-zA-z]:($|\\))/.test(result))
+						if(/^(\/|[a-zA-Z]:($|\\))/.test(result)) {
```

Comment:
did linting call for the addition of braces? if not, remove them to maintain the existing code style